### PR TITLE
Anko - Update group data source

### PR DESCRIPTION
**Pull request by at-cuongcao**
#### Related issues (Required)
- Task #171 

#### Description (Required)
- [x] Update group data source

### CODE REVIEW

File: app/src/main/java/vn/asiantech/way/data/source/remote/GroupRemoteDataSource.kt [hunk 1]
```diff
@@ -134,12 +134,8 @@ class GroupRemoteDataSource : GroupDataSource {
         return result
     }
 
-    override fun removeUserFromGroup(userId: String): Single<Boolean> {
-        val result = SingleSubject.create<Boolean>()
-        HypertrackApi.instance.removeUserFromGroup(userId, BodyAddUserToGroup(null))
-                .doOnSuccess { result.onSuccess(true) }
-                .doOnError { result.onError(it) }
-        return result
+    override fun removeUserFromGroup(userId: String): Single<User> {
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/data/source/remote/GroupRemoteDataSource.kt]: WARNING:

File: app/src/main/java/vn/asiantech/way/data/source/remote/GroupRemoteDataSource.kt [hunk 2]
```diff
@@ -254,28 +250,18 @@ class GroupRemoteDataSource : GroupDataSource {
...
-                            })
-                }, {
-                    result.onError(it)
-                })
-        return result
+                    HypertrackApi.instance.addUserToGroup(ownerId, BodyAddUserToGroup(group.id))
+                            .flatMap {
+                                postGroupInfo(group)
+                            }
+                }
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/data/source/remote/GroupRemoteDataSource.kt]: WARNING:

File: app/src/main/java/vn/asiantech/way/data/source/GroupRepository.kt [hunk 3]
```diff
@@ -48,7 +48,7 @@ class GroupRepository : GroupDataSource {
         return remoteDataSource.postInvite(userId, invite)
     }
 
-    override fun removeUserFromGroup(userId: String): Single<Boolean> {
+    override fun removeUserFromGroup(userId: String): Single<User> {
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/data/source/GroupRepository.kt]: WARNING:

**Comment by at-vinhhuynh** [comment]
@at-tienhoang 
I finish review this PR, please check it again.
thank you!

**Pull request merged**